lib/types.ts

```diff
@@ -28,9 +28,12 @@ export interface ActivityData {
   last_updated?: string;
   activity: Activity[];
   open_prs: OpenPr[];
-  pr_stale: number;
+  pr_stale?: number;
```

Comment:
Could you specify the reason for making this nullable?

```suggestion
  pr_stale: number;
```